pkg/ccl/acceptanceccl/backup_test.go

```diff
@@ -1,8 +1,8 @@
 // Copyright 2017 The Cockroach Authors.
 //
-// Licensed under the Cockroach Community Licence (the "License");
```

Comment:
I'm not sure where this wording came from, I only found it in the `acceptanceccl` dir. Older stuff I guess?